terraform/projects/app-ecs-albs/main.tf

```diff
@@ -0,0 +1,145 @@
+/**
+* ## Project: app-ecs-albs
+*
+* Create ALBs for the ECS cluster
+*
+* This may be merged back in to the nodes project
```

Comment:
@deanwilson does this comment still stand?

Reply:
Probably not anymore.